modules/org-noter-org-roam.el

```diff
@@ -30,6 +30,11 @@
   (condition-case nil
       (require 'org-roam-node)
     (error (message "`org-roam-node is not found. org-noter's org-roam support requires org-roam to be installed. Please install org-roam."))))
+;; run time warning
+(condition-case nil
+    (require 'org-roam-node)
+  (error (message "`org-roam-node is not found. org-noter's org-roam support requires org-roam to be installed. Please install org-roam.")))
```

Comment:
With this patch applied, there is a runtime message when calling `(org-noter-enable-org-roam-integration)`.

However, since that function adds a hook which needs `org-roam` to work, this might not be the desirable behavior.